src/vivarium/framework/logging/manager.py

```diff
@@ -39,22 +39,22 @@ def _terminal_logging_not_configured() -> bool:
         # fragile since it depends on a loguru's internals as well as the stability of code
         # paths in vivarium, but both are quite stable at this point, so I think it's pretty,
         # low risk.
-        return 1 not in logger._core.handlers
+        return 1 not in logger._core.handlers  # type: ignore[attr-defined]
```

Comment:
I guess loguru's private stuff isn't typed (or maybe it's mypy that can't follow it?). Regardless, I couldn't get at the handlers any other way so just ignore.